diff --git a/nevow/__init__.py b/nevow/__init__.py
index 45ac0a7..ebfb517 100644
--- a/nevow/__init__.py
+++ b/nevow/__init__.py
@@ -65,7 +65,8 @@ def _versions():
     return __version__, __version_info__, version
 
 
-__version__, __version_info__, version = _versions()
+from twisted.python.versions import Version
+__version__, __version_info__, version = "0.14.4.dev1", (0, 14, 4), Version("nevow", 0, 14, 4, dev=1)
 
 import sys
 from twisted.python.components import registerAdapter
diff --git a/setup.py b/setup.py
index b28f09a..dd97408 100644
--- a/setup.py
+++ b/setup.py
@@ -21,8 +21,7 @@ for (dirpath, dirnames, filenames) in os.walk("doc"):
 if __name__ == '__main__':
     setup(
         name='Nevow',
-        version=versioneer.get_version(),
-        cmdclass=versioneer.get_cmdclass(),
+        version="0.14.4.dev1",
         packages=find_packages(),
         py_modules=["twisted.plugins.nevow_widget"],
         include_package_data=True,
